libs/components/core/src/lib/modules/content-info-provider/descriptor.ts

```diff
@@ -0,0 +1,4 @@
+export type SkyDescriptor = {
```

Comment:
I suggest using `SkyContentInfoDescriptor` so its obvious that it's for use with `SkyContentInfo`. Same with `SkyContentInfoDescriptorType`.